src/renderer/features/multisig-operations/model/status.ts

```diff
@@ -0,0 +1,86 @@
+import { type ApiPromise } from '@polkadot/api';
+import { combine, createStore, sample } from 'effector';
+import { debounce } from 'patronum';
+
+import { type ChainId } from '@/shared/core';
+import { createFeature } from '@/shared/effector';
+import { nullable } from '@/shared/lib/utils';
+import { type AccountId } from '@/shared/polkadotjs-schemas';
+import { networkModel, networkUtils } from '@/entities/network';
+import { walletModel, walletUtils } from '@/entities/wallet';
+
+const $trigger = createStore<string>('');
+const $debouncedApis = createStore<Record<ChainId, ApiPromise>>({});
+
+sample({
+  clock: debounce(networkModel.$apis, 2000),
+  source: networkModel.$chains,
+  fn: (chains, apis) => {
+    const multisigChains = Object.values(chains)
+      .filter((chain) => networkUtils.isMultisigSupported(chain.options))
+      .map((c) => c.chainId);
+
+    return multisigChains.filter((chainId) => apis[chainId]).join(',');
```

Comment:
We can include this filter into previous one 
```javascript
networkUtils.isMultisigSupported(chain.options) && apis[chainId]
```
And use variables for readability